features/openhab-addons-external/src/main/resources/conf/openhabcloud.cfg

```diff
@@ -1,19 +0,0 @@
-############################## openHAB Cloud Connector #############################
```

Comment:
why is this class removed?

Reply:
This is part of the 2.5.x addons, which is auto-ported to 3.0.